packages/data-broker/src/services/dhis/DhisService.js

```diff
@@ -3,12 +3,7 @@
  * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
```

Comment:
Also, there's a comment in the card:
>We should:
>- Remove any client code and redundant DB data related to the above (eg legacy_report.data_services)

I left all the data alone because it's legacy and going away soon, happy with that? Or do you think we should clean it up?

Reply:
Hahaha well wether it's going away soon or not is questionable 😉 but I'm cool with leaving it on the legacy data builders 👍